projects/storefrontlib/src/cms-components/myaccount/reset-password/reset-password-form.component.ts

```diff
@@ -16,17 +11,21 @@ import { CustomFormValidators } from '../../../shared/utils/validators/custom-fo
 export class ResetPasswordFormComponent implements OnInit, OnDestroy {
```

Comment:
Breaking changes to document:
- removed submited

Reply:
Done